docs/versioned/spark/spark-structured-streaming.md

```diff
@@ -44,7 +44,7 @@ val df = spark.readStream
 ```
 
 {{< hint warning >}}
-Iceberg only supports reading data from append snapshots. Overwrite snapshots cannot be processed and will cause an exception. Similarly, delete snapshots will cause an exception by default, but deletes may be ignored by setting `streaming-skip-delete-snapshots=true`.
+Iceberg only supports reading data from snapshots of type `Append` or `Replace`. Snapshots of type `Delete` & `Overwrite` cannot be processed and will cause an exception. The spark options `streaming-skip-delete-snapshots=true` & `streaming-skip-overwrite-snapshots=true` may be used while reading from the iceberg tables where, there are known cases of deletes and overwrites performed on the table that can be ignored.
```

Comment:
Nit: The wording is a little weird for me in the last sentence.

I'd maybe say "The spark options `streaming-skip-delete-snapshots` and `streaming-skip-overwrite-snapshots` may be set to true to skip snapshots of type `Delete` and `Overwrite`, respectively, in cases where it's acceptable to ignore the table changes from those snapshots."

I'll leave that up to you though.

Reply:
Also, maybe don't use `&` (though if use that elsewhere in the docs then by all means ignore my comment 🙂  )